packages/react-codesandboxer/README.md

```diff
@@ -72,11 +72,11 @@ Children also receives the `sandboxId` and the `sandboxUrl` once those exist.
 
```

Comment:
Ahh, it ran prettier here. I can remove the prettifying if you prefer that!

Reply:
I'm not fussed about removing the prettying. All good here.